packages/visual-editor/src/components/Dropzone/EditorBlock.tsx

```diff
@@ -94,6 +94,42 @@ export const EditorBlock: React.FC<EditorBlockProps> = ({
     });
   };
 
+  if (isComponentMissing || !definition || !elementToRender) {
+    const renderMissingComponentPlacehoder = () => (
+      <div
+        style={{
+          border: '1px solid red',
+          width: '100%',
+          height: '100%',
+        }}>
+        Missing component &apos;{node.data.blockId}&apos;
+      </div>
+    );
+
+    return (
+      <DraggableComponent
+        placeholder={placeholder}
+        id={node.data.id}
+        index={index}
+        isAssemblyBlock={isAssembly || isAssemblyBlock}
+        isDragDisabled={false}
+        isSelected={selectedNodeId === node.data.id}
+        userIsDragging={userIsDragging}
+        isContainer={isContainer}
+        blockId={node.data.blockId}
+        coordinates={coordinates!}
+        wrapperProps={wrapperProps}
+        onClick={onClick}
+        onMouseOver={onMouseOver}
+        displayName={displayName || definition?.name}>
+        {renderMissingComponentPlacehoder()}
+        {/* {isStructureComponent && userIsDragging && (
+          <Hitboxes parentZoneId={zoneId} zoneId={componentId} isEmptyZone={isEmptyZone} />
+        )} */}
```

Comment:
Might be better to delete this commented out code

Reply:
✅  FIXED 65ad695c0ba70b01158c1492491aaa947127b931